tests/commands/autoupdate_test.py

```diff
@@ -263,6 +263,27 @@ def test_does_not_reformat(tmpdir, out_of_date, store):
     assert cfg.read() == expected
 
 
+def test_does_not_change_mixed_endlines(up_to_date, tmpdir, store):
+    """Test that line endings are not being changed."""
```

Comment:
this docstring is redundant with the test name, I would remove it

Reply:
I like docstrings in every method. But I see your point here. Removed